src/main/java/org/folio/marc/migrations/config/SpringBatchConfig.java

```diff
@@ -192,4 +192,34 @@ public Step remapAuthRetryRecordsStep(JobRepository jobRepository,
         .stepOperations(customThrottler)
         .build();
   }
+
+  @Bean("remappingRetrySaveJob")
+  public Job remappingRetrySaveJob(JobRepository jobRepository,
+                              @Qualifier("remapRetrySaveAuthRecordsStep") Step remapAuthRecordsStep) {
+    return new JobBuilder("remappingRetrySave", jobRepository)
+        .incrementer(new RunIdIncrementer())
+        .start(remapAuthRecordsStep)
+        .build();
+  }
+
+  @Bean(name = "remapRetrySaveAuthRecordsStep")
```

Comment:
Should we remove "Auth" from the name? I assune this will aslo work for instances